lib/avromatic/model/attributes.rb

```diff
@@ -154,8 +164,9 @@ def avro_field_options(field)
           options
         end
 
-        def add_serializer(field)
-          custom_type = Avromatic.type_registry.fetch(field)
+        def add_serializer(field, field_class)
+          # TODO: handle serializer for custom type within union
```

Comment:
Similar question as earlier about raising an error if someone tries to do something we don't yet support